README.md

```diff
@@ -89,6 +89,8 @@ Set environment variables with credentials to Voucherify and commercetools APIs.
     - (optional) `DEBUG_STORE_REQUESTS_IN_JSON` - `true` if you want to keep external requests / response in a JSON file.
     - (optional) `DEBUG_STORE_REQUESTS_DIR` - name of the directory where JSON files with request / responses are stored.
 
+[*] Due to the fact that in the commercetools there can be a lot of possible prices per one product, but in Voucherify there is only one, it is needed to choose proper price. You can find proper values by searching in `*any product* > Variants > *any variant* > Prices` for row which have proper price scope for your project. Depends on sitiuation you may need to use all of four variables or less, but at least the `COMMERCE_TOOLS_PRODUCTS_CURRENCY` variable is required. In case when you does not choose right values, then some functionallities won't work properly (e.g. adding free item with coupon). [Here](https://docs.commercetools.com/api/projects/products#price-selection) you can read more about commercetools mechanism called "Price Selection" which we used. 
+
```

Comment:
Due to the fact that in the commercetools can be a lot of possible prices per one product, but in Voucherify there is only one, it is needed to choose the proper price. You can find proper values by searching in `*any product* > Variants > *any variant* > Prices` for rows which have proper price scope for your project. Depending on the situation you may need to use all of four variables or less, but at least the `COMMERCE_TOOLS_PRODUCTS_CURRENCY` variable is required. In case when you do not choose the right values, then some functionalities won't work properly (e.g. adding free item with coupon). [Here](https://docs.commercetools.com/api/projects/products#price-selection) you can read more about commercetools mechanism called "Price Selection" which we used.

A few language fixes. Pls check if everything make sense in my example proposition.